controller/konnect/ops/sdkfactory_mock.go

```diff
@@ -41,16 +53,16 @@ func (m MockSDKWrapper) GetMeSDK() MeSDK {
 }
 
 type MockSDKFactory struct {
-	w *MockSDKWrapper
+	Wapper *MockSDKWrapper
```

Comment:
- `wapper` != `wrapper`.
 - can we just call this `sdk` (or similar)? I believe it would be easier to follow if we didn't use "wrapper" everywhere but just "sdk" to be returned. WDYT?

Reply:
done.